src/helpers/agent-setup.sh

```diff
@@ -17,6 +17,8 @@ function ensure_agent_ownership {
   chown -R vcap:vcap "$JOB_DIR/packages/dd-agent/"
   # make sure that root owns the system probe config
   chown root:root "$JOB_DIR/config/system-probe.yaml" && chmod 0644 "$JOB_DIR/config/system-probe.yaml"
+  # add dd-agent user as an alias of vcap
+  sudo useradd -N -r -s /usr/sbin/nologin -u $(id -u vcap) -g vcap -o dd-agent
```

Comment:
It looks like this function gets called on each process start? If so, does it handle if the `dd-agent` user already exists?

Reply:
I guess it does now.